bus_sdk/publishers/event_publisher.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from bus_sdk.messages import Message
+
+
+class EventPublisher(ABC):
+    @property
+    @abstractmethod
+    def name(self) -> str:
```

Comment:
Doesn't feel needed?

Reply:
It requires a name so that it can map the types to classes which is then used by the factory function:

```
def get_publisher_types() -> dict:
    return {pub.name: pub for pub in PUBLISHERS}
```